From e8bb4970b07f024b626e2ae511ce282e318de427 Mon Sep 17 00:00:00 2001
From: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Date: Wed, 18 Dec 2019 21:44:53 +0100
Subject: [PATCH 09/18] arm: dts: imx7-colibri: usb device aka gadget specific
 device tree

USB device aka gadget specific device trees for the Toradex Easy
Installer.

Signed-off-by: Marcel Ziswiler <marcel.ziswiler@toradex.com>
---
 arch/arm/boot/dts/imx7-colibri.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7-colibri.dtsi b/arch/arm/boot/dts/imx7-colibri.dtsi
index c551aa1fc534..b5852091b85b 100644
--- a/arch/arm/boot/dts/imx7-colibri.dtsi
+++ b/arch/arm/boot/dts/imx7-colibri.dtsi
@@ -509,7 +509,7 @@
 };
 
 &usbotg1 {
-	dr_mode = "host";
+	dr_mode = "peripheral";
 };
 
 &usdhc1 {
-- 
2.25.1

